presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergTableProperties.java

```diff
@@ -114,4 +125,9 @@ public static String getFormatVersion(Map<String, Object> tableProperties)
     {
         return (String) tableProperties.get(FORMAT_VERSION);
     }
+
+    public static String getDeleteMode(Map<String, Object> tableProperties)
+    {
+        return (String) tableProperties.get(DELETE_MODE);
+    }
```

Comment:
Seems table property `delete_mode ` has no use. The same as above, maybe we should firstly figure out whether the `delete-mode` be a connector property or a table property.